lab4.md

```diff
@@ -158,6 +158,71 @@ In Python code you'd type in `os.getenv("Http_X_Output_Mode")`.
 
 You can see that all other HTTP context is also provided such as `Content-Length` when the `Http_Method` is a `POST`, the `User_Agent`, Cookies and anything else you'd expect to see from a HTTP request.
 
+### Test combined output
+ 
+By default function `stdout/stderr` is combined. You have an option to write `stderr` to the container logs and use `stdout` for function response using the `combine_output` flag.
+
+To test the flag create a new function using the dockerfile language:
+
+```
+faas new --lang dockerfile --prefix <your-docker-username-here> combout
```

Comment:
`combout` isn't meaningful.

Please use a new or existing Python 3 example. Make the output something that is sensitive to formatting like a JSON response.

Show how adding logging into the response breaks the JSON response.

Add some lines like:

```
sys.stderr.write("Started the function")
```


```
sys.stderr.write("Got to this point in the function")
```

Show how they come out in the function output normally and then show how they can be separated out with the flag.